content/docs/hello-world.md

```diff
@@ -15,36 +15,36 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Nó hiển thị một tiêu đề với dòng chữ "Hello, world!" trên trang web.
```

Comment:
Từ `It` => `nó` em nghĩ nên chuyển thành "Ví dụ trên"

Reply:
Good point!